app/cdap/components/RecipeDetails/utils.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+export const dateFormatting = (millisecondsTime) => {
```

Comment:
plz rename this function based on the behavior
it is accepting time or date? if date is accepted in milleseconds then millisecondsTime should be called dateInMilliseconds
and what is the format it returns, i think we have some methods on Date that can return the expected format
if not available on Date methods then see if lodash or moment are used in this app
if none are there, then plz inform me here

also add JSDOC noted plz, this is mandatory